db/sql/00_msar.sql

```diff
@@ -4775,10 +4775,15 @@ END;
 $$ LANGUAGE plpgsql;
 
 
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, anyelement);
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, anyelement, boolean);
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, text, boolean);
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, numeric, boolean);
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, anycompatible, boolean);
```

Comment:
```suggestion
DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, anyelement, boolean);
```

Reply:
@Anish9901 and I chatted about this on a call and agreed on the approach in 08cd6abf8217b762c6c5a81f400ddf9191a4d015